unittests/test_flush_auditlog.py

```diff
@@ -0,0 +1,45 @@
+from dojo.tasks import flush_auditlog
+from .dojo_test_case import DojoTestCase
+from django.conf import settings
+from auditlog.models import LogEntry
+from datetime import date, datetime
+from dojo.models import Finding
+from dateutil.relativedelta import relativedelta
+import logging
+logger = logging.getLogger(__name__)
+
+
+class TestFlushAuditlog(DojoTestCase):
+    fixtures = ['dojo_testdata.json']
+
+    def test_flush_auditlog_disabled(self):
+        settings.AUDITLOG_FLUSH_RETENTION_PERIOD = -1
```

Comment:
```suggestion
    @override_settings(AUDITLOG_FLUSH_RETENTION_PERIOD = -1)
    def test_flush_auditlog_disabled(self):
```
This is better practice for these scenarios 
https://docs.djangoproject.com/en/5.0/topics/testing/tools/#django.test.override_settings

Reply:
Alright I will keep this in mind for the future ;)